python/lsst/pipe/base/pipelineIR.py

```diff
@@ -52,6 +52,18 @@
 from lsst.utils.introspection import find_outside_stacklevel
 
 
+class PipelineSubsetCtrl(enum.Enum):
```

Comment:
It would be nice to add a class docstring here, to override the default Enum docstring that gets inherited instead.